doc/sections/ex-gwt-mt3dms-p04.tex

```diff
@@ -0,0 +1,51 @@
+\section{Two-Dimensional Transport in a Diagonal Flow Field (MT3DMS Example Problem 4)}
+
+The third demonstrated MT3DMS-\mf transport comparson is for a two-dimensional transport in a diagonal flow field. This problem is similar to the preceding problem with two important changes. First, the flow direction is now oriented at a 45-degree angle relative to rows and columns of the numerical grid. Owing the use of MT3DMS for comparison, the \mf solution uses the traditional DIS package (not DISU or DISV).  The second notable change is that the number of rows and columns has been expanded in order to accomodate a longer simulation period of 1,000 days. Because of the orientation of the flow field relative to the model grid, and the sharpness of the migrating concentration front, this test problem presents a challenging set of conditions to simulate. Three scenarios test alternative advection formulations, as summarized in table~\ref{tab:ex-gwt-mt3dms-p04-scenario}
+
+% add scenario table
+\begin{ScenarioTable}{
+       Variation in advection scheme for testing transport solutions. The $mixelm$ parameter is an integer flag used by the model to set the advection solution option.  A value of -1 uses a third-order TVD scheme \citep{zheng1999mt3dms}, 0 uses the standard finite-difference (FD) solution, and 1 invokes the method-of-characteristics (MOC) solution in MT3DMS.  Because MF6 does not have a method-of-characteristics solution for transport, a value of 1 is the same as specifying 0 (FD solution)}
+       {tab:ex-gwt-mt3dms-p04-scenario}
+       {../tables/ex-gwt-mt3dms-p04-scenario}
+\end{ScenarioTable}
+
+Model parameter values for this problem are provided in table~\ref{tab:ex-gwt-mt3dms-p0401}.
+
+% add 2nd static parameter value table
+\begin{StandardTable}
+	{Hydraulic and transport model parameter values used in the two-dimensional transport in a diagonal flow field problem.}
+	{tab:ex-gwt-mt3dms-p0401}
+	{../tables/ex-gwt-mt3dms-p0401}
+\end{StandardTable}
+
+The same analytical solution used in the previous problem can be used for this problem after applying the necessary updates to select parameters - most noteably the dispersion and porosity terms and that an inter-model comparison is drawn after 1,000 days instead of 365 days. Figure 36 in \cite{zheng1999mt3dms} shows four different solutions for this problem: (1) analytical, (2) Method of Characteristics (MOC), (3) upstream finite difference (FD), and (4) Total Variation Dimishing (TVD) or "ULTIMATE" scheme.  Both the MOC and TVD solutions demonstrate a reasonable agreement with the analytical solution. However, the upstream finite difference solution reflects considerably more spread from simulation of too much dispersion - in this case numerical dispersion instead of hydrodynamic dispersion. 
```

Comment:
the word ULTIMATE is surrounded by double quotes, which can't be used with latex.  You need to use ``ULTIMATE''